src/cli/upsert.rs

```diff
@@ -78,26 +111,36 @@ async fn run_upsert(
         }
     }
 
+    let mut arg_id = String::from("");
     if let Some(id) = args.nth(0) {
-        ids.push(id.as_string()?);
+        arg_id = id.as_string()?;
+    }
+
+    if arg_id != "" {
+        let mut arg_content = String::from("");
+        if let Some(content) = args.nth(1) {
+            arg_content = content.as_string()?;
+        }
+
+        // An empty value is a legitimate document
+        rows.insert(arg_id, arg_content);
     }
 
     let bucket = state.active_cluster().cluster().bucket("travel-sample");
     let collection = bucket.default_collection();
 
-    debug!("Running kv get for docs {:?}", &ids);
+    debug!("Running kv get for docs {:?}", &rows);
 
     let mut results = vec![];
-    for id in ids {
-        match collection.get(&id, GetOptions::default()).await {
-            Ok(res) => {
+    for (id, content) in rows.iter() {
+        match collection
+            .upsert(id, content, UpsertOptions::default())
```

Comment:
I see you are storing the content as a string, but right now I'm not sure if the upsert thinks (simple code in couchbase-rs) that is a string type and will double encode it? Maybe we need to (even if wasteful) for now turn it into a JsonValue with serde again before inserting.. if it looks fine on the server then all good